be/test/storage/lake/compaction_task_test.cpp

```diff
@@ -52,6 +52,15 @@ class LakeCompactionTest : public TestBase, testing::WithParamInterface<Compacti
             ASSERT_TRUE(dynamic_cast<VerticalCompactionTask*>(task.get()) != nullptr);
         }
     }
+
+protected:
+    void SetUp() override {
+        config::enable_size_tiered_compaction_strategy = GetParam().enable_size_tiered_compaction_strategy;
```

Comment:
need restore the configvar to default value, in case running case sequentially, it could break other cases by the change of configvar value other than default.

Reply:
done